Makefile

```diff
@@ -254,7 +255,8 @@ restore-generated-framework-info:
 
 restore-generated-file: restore-generated-framework-info
 	@echo "restore application info"
-	@echo -e "package config\n\nconst LastCommitLog = \"N/A\"\nconst BuildDate = \"N/A\"" > config/generated.go
+	@echo -e "package config\n\nconst LastCommitLog = \"N/A\"" > config/generated.go
```

Comment:
seems you changed the format, that's fine, but remember to update the geneted file in other projects.